waku/v2/protocol/waku_swap/waku_swap.nim

```diff
@@ -242,6 +248,8 @@ proc init*(wakuSwap: WakuSwap) =
     else:
       info "Payment threshhold not hit"
 
+    discard wakuSwap.logAccountMetrics(peerId)
```

Comment:
We try to avoid using `discard` to spawn futures (it could suppress errors) - you should use `asyncSpawn` instead. Admittedly there are still a number of places where `discard` is used similarly, but ideally this will slowly be refactored. Reasoning here: https://github.com/status-im/nim-waku/issues/146